02-install/02-01-install_rust_on_linux.md

```diff
@@ -1,50 +1,52 @@
 #Rust for Linux
 
-Rust 支持主流的操作系统，Linux,Mac和 windows。
+Rust supports mainstream operating systems，Linux,Mac and windows.
 
-Rust 为Linux用户提供了两种安装方式：
+Rust two installation methods for Linux users：
 
-### 1、直接下载安装包：
+### 1、Download the installation package directly：
 
-直接下载安装包的话需要检查一下你当前操作系统是64位还是32位，分别下载对应的安装包。
+To download the installation package directly, you need to check whether your current operating system is 64-bit or 32-bit, and download the corresponding installation package.
 
-查看操作系统请在终端执行如下命令:
+To view the operating system, execute the following command in the terminal:
 
 `uname -a`
 
-结果如下图所示：
+The result is shown below：
 
 ![check system info](../image/01-install-on-linux-check-system.png)
 
-如上图所示，如果是 **x86_64** 则证明是64位系统，需要[下载](https://static.rust-lang.org/dist/rust-1.5.0-x86_64-unknown-linux-gnu.tar.gz)64位安装包；
+As shown above, if it is **x86_64** 64-bit system，to [download](https://static.rust-lang.org/dist/rust-1.5.0-x86_64-unknown-linux-gnu.tar.gz)64-bit installation package；
```

Comment:
maybe the word must have space charactor between `download` and `64-bit` ?

Reply:
done